docs/en/query_language/functions/uuid_function.md

```diff
@@ -43,7 +50,7 @@ UUID.
 └──────────────────────────────────────┘
 ```
 
-Note that the String value has to correspond to the UUID format.
+The `toUUID()` function returns an error if the String value does not correspond to the UUID format. For example:
 
 ``` sql
 :) SELECT toUUID('61f0c4045cb3-11e7907ba6006ad3dba0') AS uuid
```

Comment:
this example leads to `DB::Exception: Cannot parse uuid 61f0c4045cb3-11e7907ba6006ad3dba0: Cannot parse UUID from String.`